### PR TITLE
[SW-527] Fix scrollbars visibility in firefox

### PR DESCRIPTION
**Pull request by peronczyk**

### CODE REVIEW

File: src/popup/components/Scrollable.vue [hunk 1]
```diff
@@ -7,7 +7,7 @@
 </template>
 
 <style lang="scss" scoped>
-@use '../../styles/variables';
+@use '../../styles/variables' as *;
```
Comment (pstachel) [inline src/popup/components/Scrollable.vue:10]: If the whole project we use variables as `variable.$color-some-color` should we diverge from it in favour of using just a variable name?

**Review by pstachel — APPROVED** [review]

File: src/popup/components/Scrollable.vue (hunk 1 above)
Comment (peronczyk) [reply src/popup/components/Scrollable.vue:10]: We are not using any 3rd party frameworks so in my opinion it's easier to not use the `viariables` prefix. We can gradually migrate into this approach.

**Review by konradodo — APPROVED** [review]

**Review by CedrikNikita — DISMISSED** [review]
Test ability to merge by adding a commit as a request change.

File: src/popup/App.vue [hunk 2]
```diff
@@ -17,7 +17,7 @@
       v-show="!qrScannerOpen"
       ref="innerElement"
       class="app-inner"
-      :class="{ 'show-scrollbar': $route.meta.showScrollbar }"
+      :class="{ 'show-scrollbar': showScrollbar }"
```
Comment (CedrikNikita) [inline src/popup/App.vue:20]: Test ability to merge by adding a commit as a request change.

**Review by CedrikNikita — CHANGES_REQUESTED** [review]

**Pull request merged**